const cl_type get() type qualifiers warning

When compiling with this parameters `-Wall -Wextra -std=c++11 -O2 -Os -s -DNDEBUG`, I got 

> cl2.hpp:1829:5: warning: type qualifiers ignored on function return type [-Wignored-qualifiers]
>      const cl_type get() const { return object_; }

--- a/include/CL/cl2.hpp
+++ b/include/CL/cl2.hpp
@@ -1805,9 +1805,6 @@ public:
 
     cl_type get() const { return object_; }
 
-    cl_type get() { return object_; }
-
-
 protected:
     template<typename Func, typename U>
     friend inline cl_int getInfoHelper(Func, cl_uint, U*, int, typename U::cl_type);

--- a/include/CL/cl2.hpp
+++ b/include/CL/cl2.hpp
@@ -1803,7 +1803,7 @@ public:
 
     cl_type& operator ()() { return object_; }
 
-    const cl_type get() const { return object_; }
+    cl_type get() const { return object_; }
 
     cl_type get() { return object_; }
 
@@ -8383,7 +8383,6 @@ public:
         const vector<Event>* events = NULL,
         Event* event = NULL) const
     {
-        cl_event tmp;
         cl::vector<void*> svmRawPointers;
         svmRawPointers.reserve(svmPointers.size());
         for (auto p : svmPointers) {
@@ -8421,7 +8420,6 @@ public:
         const vector<Event>* events = NULL,
         Event* event = NULL) const
     {
-        cl_event tmp;
         cl::vector<void*> svmRawPointers;
         svmRawPointers.reserve(svmContainers.size());
         for (auto p : svmContainers) {

